src/core/components/Root/index.js

```diff
@@ -3,27 +3,40 @@ import * as React from 'react';
 import { ConnectedRouter } from 'connected-react-router';
 import { CookiesProvider, Cookies } from 'react-cookie';
 import { Provider } from 'react-redux';
+import config from 'config';
 
 import I18nProvider from 'core/i18n/Provider';
 import type { I18nType } from 'core/types/i18n';
 import type { ReduxStore } from 'core/types/redux';
 import type { ReactRouterHistoryType } from 'core/types/router';
 
 type Props = {|
+  _config?: typeof config,
   children: React.Node,
   cookies?: typeof Cookies,
   history: ReactRouterHistoryType,
   i18n: I18nType,
   store: ReduxStore,
 |};
 
-const Root = ({ children, history, i18n, store, cookies = null }: Props) => (
+const Root = ({
+  _config = config,
+  children,
+  history,
+  i18n,
+  store,
+  cookies = null,
```

Comment:
I didn't want to touch this without asking, but should `cookies` go after `children` if doing ordering?